src/core/componentSystems/PhysicsSystem.js

```diff
@@ -116,7 +120,13 @@ export class PhysicsSystem extends MRSystem {
         if (entity instanceof MRDivEntity) {
             this.initPhysicsBody(entity);
             this.registry.add(entity);
-            entity.addEventListener('modelchange', this.entityEventUpdate);
+            entity.addEventListener('modelchange', (e) => {
+                this.entityEventUpdate(e);
+            });
+
+            entity.addEventListener('child-updated', (e) => {
```

Comment:
if child-updated is one of our own events, it would be better as `childupdated` since we're switching away from the hyphenated ones

Reply:
agreed.

Reply:
I think I might change it to entityupdated?

Reply:
that works 👍